src/pages/featured-globe/featured-globe-component.jsx

```diff
@@ -99,19 +102,40 @@ function FeaturedGlobe({
           <GlobePageIndicator onMouseEnter={() => setActiveGlobesMenu(true)} />
         )}
 
-        {selectedFeaturedMap && (
-          <SelectedFeaturedMapCard
-            className={cx(uiStyles.uiTopLeft, {
-              [uiStyles.blur]: activeGlobesMenu && !selectedFeaturedPlace,
-            })}
-            selectedFeaturedMap={selectedFeaturedMap}
-            selectedSidebar={selectedSidebar}
-            isFullscreenActive={isFullscreenActive}
-            selectedFeaturedPlace={selectedFeaturedPlace}
-            spinGlobe={spinGlobe}
-            handle={spinGlobeHandle}
-          />
-        )}
+        <div className={cx(uiStyles.uiTopLeft, uiStyles.featuresContainer)}>
```

Comment:
I would add a overflow-y-auto so the user can still access the second card on short screens